thrift-analytics-publisher/src/main/java/org/wso2/am/thrift/analytics/publisher/ThriftMetricEventBuilder.java

```diff
@@ -148,6 +149,8 @@ private void remapAndAddAttribute(String key, Object value) {
         } else if (key.equals(REQUEST_TIMESTAMP)) {
             String timestampString = (String) value;
             eventMap.put(REQUEST_TIMESTAMP, OffsetDateTime.parse(timestampString).toInstant().toEpochMilli());
+        } else if (APPLICATION_ID.equals(key)) {
+            // Ignore application UUID, as the application ID integer will be added through properties
         } else {
             eventMap.put(getRenamedKey(key), value);
         }
```

Comment:
@Avishka-Shamendra shall we optimize this as:
```suggestion
        } else if (!APPLICATION_ID.equals(key)) {
            // Ignore application UUID, as the application ID integer will be added through properties
            eventMap.put(getRenamedKey(key), value);
        }
```